platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependenciesExtensions.kt

```diff
@@ -97,6 +108,14 @@ class DependenciesExtensions {
 fun Dependencies.module(group: String?, name: String, version: String?): ExternalModuleDependency = module(group, name, version)
 
 
+/**
+ * Creates a dependency constraint based on the group, name and version (GAV) coordinates.
+ *
+ * @since 8.7
+ */
+fun Dependencies.constraint(group: String?, name: String, version: String?): DependencyConstraint = constraint(group, name, version)
```

Comment:
Can you explain what this extension does? It looks like it adds an extension that calls itself?

Reply:
Yes, it's for named arguments in Kotlin, which don't work for Java methods even if they have parameter names. I'll add a comment to this and the other extension with the same style.